Core/HLE/sceUmd.cpp

```diff
@@ -495,7 +504,9 @@ void __UmdReplace(std::string filepath) {
 		}
 	}
 	delete currentUMD;
-
+	UMD_insterted = false;
+	sleep_ms(200); // Wait sceUmdCheckMedium call
```

Comment:
Hm, this isn't very nice .. on the other hand, this will probably be pretty reliable as long as __UmdReplace is not called from the emulator thread. I fear though that it may be, on Android? Got to check to be sure though.